### PR TITLE
Fixes #2004. TextFormatter.Justified not adding the extra spaces.

### LINKED ISSUE #2004: TextFormatter.Justify method doesn't add the extra spaces.

### PR DESCRIPTION
**Pull request by BDisp**
Fixes #2004 - When justifying a text the last char mus always be equal to the last char of the original text. The extra spaces were decremented without adding the spaces.

## Pull Request checklist:

- [x] I've named my PR in the form of "Fixes #issue. Terse description."
- [x] My code follows the [style guidelines of Terminal.Gui](https://github.com/gui-cs/Terminal.Gui/blob/develop/.editorconfig) - if you use Visual Studio, hit `CTRL-K-D` to automatically reformat your files before committing.
- [x] My code follows the [Terminal.Gui library design guidelines](https://github.com/gui-cs/Terminal.Gui/blob/develop/CONTRIBUTING.md)
- [x] I ran `dotnet test` before commit
- [ ] I have made corresponding changes to the API documentation (using `///` style comments)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any poor grammar or misspellings
- [x] I conducted basic QA to assure all features are working

### CODE REVIEW

File: UnitTests/CheckboxTests.cs [hunk 1]
```diff
@@ -327,44 +335,56 @@ public void TextAlignment_Justified ()
...
-			Assert.Equal (new Size (25, 1), checkBox.TextFormatter.Size);
-			Assert.Equal ("Check this out 你", checkBox.Text);
-			Assert.Equal ("╴ Check this out 你", checkBox.TextFormatter.Text);
-			Assert.False (checkBox.AutoSize);
 
 			var expected = @"
 ┌ Test Demo 你 ──────────────┐
 │                            │
-│ ╴  Check  this  out  你    │
+│ ╴   Check  first  out  你  │
```
Comment (tig) [inline UnitTests/CheckboxTests.cs:361]: I don't understand why the extra space here (before `Check`) makes sense. 

What am i missing???

**Review by tig — CHANGES_REQUESTED** [review]
Why is this needed? See my comment.

File: UnitTests/CheckboxTests.cs (hunk 1 above)
Comment (BDisp) [reply UnitTests/CheckboxTests.cs:361]: The checkbook has a justify alignment option and that why is displaying this format.

Comment (tig) [reply UnitTests/CheckboxTests.cs:361]: Got it. Missed that. Thanks.

**Pull request merged**